### PR TITLE
Brand Refresh: The Bleeding Edge

### PR DESCRIPTION
**Pull request by lal65**
# Description:
Still Work In Progress - you can review if you want though :+1:

### CODE REVIEW

**Review by lal65 — COMMENTED** [review]
Needs some changes and follow-ups with design.

**Review by kjl16 — APPROVED** [review]

File: packages/navigation-tile-image/package.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  "name": "@psu-ooe/navigation-tile-image",
+  "version": "1.0.0",
+  "description": "Provides a navigation tile component that supports an image.",
+  "publishConfig": {
+    "access": "public",
+    "registry": "https://npm.pkg.github.com/"
+  },
+  "dependencies": {
+    "@psu-ooe/base": "^2.0",
+    "@psu-ooe/sprite": "^2.0"
```
Comment (lal65) [inline packages/navigation-tile-image/package.json:11]: Drop the dependency.

File: packages/navigation-tile/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
```
Comment (lal65) [inline packages/navigation-tile/package.json:1]: Let's drop `sprite` from dependencies.

**Review by lal65 — COMMENTED** [review]
Recent commits LGTM.

**Review by kjl16 — APPROVED** [review]

**Pull request merged**